pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/block/base/PylonInventoryBlock.kt

```diff
@@ -65,6 +65,16 @@ abstract class PylonInventoryBlock(block: Block) :
         }
     }
 
+    fun getItems() : List<ItemStack> {
+        val items = mutableListOf<ItemStack>()
+        for(inv in inventories){
+            for(item in inv.items){
+                item?.let(items::add)
+            }
+        }
+        return items
```

Comment:
```suggestion
        return inventories.flatMap { it.items }
```
Secret Kotlin cult technique, do not speak of it

Reply:
Doesn't compile because the items are nullable